app/controllers/home_controller.rb

```diff
@@ -5,24 +5,16 @@ class HomeController < ActionController::Base
   layout 'application'
 
   def index
-    @status = Rails.cache.read("rave_status")
+    # cache is nil if not set or if expired
+    # rave_status expires after 5 minutes, goes back to off by default
+    @status = Rails.cache.read("rave_status").nil? ? "off" : Rails.cache.read("rave_status")
   end
 
   def send_alert
-    if request[:status] == "on"
-      status = "off"
-    else
-      status = "on"
-    end
+    status = request[:status] == "on" ? "off" : "on"
```

Comment:
yes the purpose of the method is to toggle the status, that's a good catch. I will change the name